src/main/java/com/microsoft/jenkins/keyvault/KeyVaultSecretStringBinding.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE file in the project root for license information.
+ */
+
+package com.microsoft.jenkins.keyvault;
+
+import hudson.Extension;
+import hudson.FilePath;
+import hudson.Launcher;
+import hudson.model.Run;
+import hudson.model.TaskListener;
+import org.apache.commons.lang3.StringUtils;
+import org.jenkinsci.Symbol;
+import org.jenkinsci.plugins.credentialsbinding.Binding;
+import org.jenkinsci.plugins.credentialsbinding.BindingDescriptor;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.io.IOException;
+
+public class KeyVaultSecretStringBinding extends Binding<SecretStringCredentials> {
```

Comment:
I think we can get this for free as SecretStringCredentials implements string credentials?

Reply:
Confirmed.

I should have checked this more carefully.